core/test/unit/config_spec.js

```diff
@@ -871,8 +869,7 @@ describe('Config', function () {
             // Run the test code
             config.checkDeprecated();
 
-            logStub.calledOnce.should.be.true;
-            logStub.calledWithMatch('mail.fromaddress').should.be.true;
+            logStub.calledOnce.should.be.false;
```

Comment:
This changes the functionality to be the opposite of what it was and what was intended - that's not OK ;)